contracts/source/TestERC20.sol

```diff
@@ -0,0 +1,49 @@
+pragma solidity 0.6.3;
+
+contract TestERC20 {
+	event Transfer(address indexed from, address indexed to, uint256 value);
+	event Approval(address indexed owner, address indexed spender, uint256 value);
+	mapping (address => uint256) public balanceOf;
+	mapping (address => mapping (address => uint256)) public allowance;
+	uint256 public totalSupply;
+	bytes public symbol;
+
+	constructor(bytes memory _symbol) public {
+		symbol = _symbol;
+	}
+	function kill() public {
+		selfdestruct(msg.sender);
+	}
+	function mint(uint256 amount) public {
+		balanceOf[msg.sender] += amount;
+	}
+	function burn(uint256 amount) public {
+		require(balanceOf[msg.sender] >= amount);
```

Comment:
This require seems unnecessary for a test ERC20.  😄

Reply:
I ended up adding it to avoid mind-boggling bugs when we underflow!